packages/cdr/docker/jsreport/css/jquery.calendars.picker.css/content.css

```diff
@@ -1,35 +1,42 @@
 /* Default styling for jQuery Calendars Picker v2.0.0. */
+

```

Comment:
@cfaife please undo the reformatting of this file. It was imported as a vendor file, it's not something that we wrote so i want to leave it untouched so that if we even need to update it, we can see what the changes are.